extensions/utils/TaskScheduler.js

```diff
@@ -0,0 +1,84 @@
+const async = require('async');
```

Comment:
I had debated this, and felt it could go either way. But I can agree with this, considering it is, in fact, still performing CRR in the end regardless of what feature (i.e. replication or lifecycle) it is doing it for.